### PR TITLE
Update legend (and a whole lot more)

### PR DESCRIPTION
**Pull request by mindreeper2420**
This branch should be pulled down and run locally before merging into master. There are installation steps in the updated README file.

### CODE REVIEW

**Comment by mindreeper2420** [comment]
After this has been verified, please let me know. The commits need to be squashed in order to keep a clean history.

File: _includes/nav.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                        <a href="https://github.com/fabric8io/fabric8-ux/issues">Issues</a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/fabric8io/fabric8-ux/milestone/5">Backlog</a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/fabric8io/fabric8-ux/milestone">Milestones</a>
+                    </li>
+                    <li>
+                        <a href="invision-links.html">Designs</a>
```
Comment (catrobson) [inline _includes/nav.html]: Can we put Designs as the first tab/default tab?

Comment (mindreeper2420) [inline _includes/nav.html]: @catrobson sure - should that be the default page?

Comment (mindreeper2420) [inline _includes/nav.html]: I've updated the navigation to have the Design tab first. The site will still load the repo information and installation instructions first.

**Pull request merged**